cli/azd/pkg/exec/runargs.go

```diff
@@ -29,6 +29,9 @@ type RunArgs struct {
 
 	// When set will call the command with the specified StdIn
 	StdIn io.Reader
+
+	// When set will call the command with the specified StdOut
+	StdOut io.Writer
```

Comment:
Please add a `WithStdOut(...)` func to be consistent with the builder pattern already in place here.

Reply:
Added..  But I don't really understand why we need this (and all the builder).  All fields are exported and can be set at any time, there's no point on having a builder there.